release/karaf/itests/switchyard/pom.xml

```diff
@@ -377,7 +377,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-deploy-plugin</artifactId>
                 <configuration>
-                    <skip>true</skip>
+                    <skip>false</skip>
```

Comment:
This is required that MEAD build (ie brew maven-build which used for product build env) doesn't allow skipping  deploy.
If not setting this, we can't build the project. So I have changed all the <skip>true</skip> to false.